web_src/js/webcomponents/overflow-menu.ts

```diff
@@ -151,6 +153,10 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
         }, 0);
       },
     });
+
+    this.tippyContent.querySelector('.item').addEventListener('click', () => {
+      this.button._tippy.hide();
+    });
```

Comment:
Doesn't seem right.

1. `querySelector` only selects the first item?
2. Does `_tippy` always exists?

(ps: I don't quite logic the logic why "item click" means "hide tippy", the tippy doesn't hide itself?)

Reply:
> https://atomiks.github.io/tippyjs/v6/all-props/#hideonclick 
> Determines if the tippy hides upon clicking the reference or **outside of the tippy**.